### DISCUSSION TITLE
たまに左右反転が起こる件について

### DISCUSSION THREAD

**Discussion by Porn44** [Yowkees/keyball#421] [Q&A] [answered]
いつもお世話になっております。
２つのKeyball39を組み立てて快適に使用させていただいております。
最初に組み立てたものについて、標題の件をお尋ねいたします。
数か月間、通常通り運用できていたのですが、ある時から左右が反転する現象が起こるようになりました。
ドライバーをアンインストールしたり、ファームウェアを更新したり、タクトスイッチを押したりすると元に戻ります。
何か考えられる原因がありますか。ご教示いただきたくお願い申し上げます。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

左右反転がたまに起こるということで、おそらく基板上に接続が不安定な半田付けがあると思います。
"LEFT"の表示がある左右判定ジャンパーと、その近くにあるダイオードの半田付けをやり直してみてください。

以上、お手数ですがご確認お願いいたします。

**Reply by Porn44** [reply]
ありがとうございます。試してみます。

**Comment by Porn44** [comment]
度々恐れ入ります。
上記の件を試してみたところ、改善されませんでした。
ちなみにOLEDはトラックボールのある右手側にロゴが表示されています。
右側のpromicroにケーブルをつなぐと正しく認識され、トラックボールも正常に動作します。

**Reply by Porn44** [reply]
申し訳ありません。左手側のすべてのダイオードを再度見直したところ、正常に動作しました。ありがとうございました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ご自身でダイオードを見直していただき解決できたようでよかったです。
ぜひ末永くご愛用いただければ幸いです。
